webApp/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,50 @@
+package com.github.jetbrains.rssreader.webApp
+
+import com.github.jetbrains.rssreader.app.FeedAction
+import com.github.jetbrains.rssreader.app.FeedState
+import com.github.jetbrains.rssreader.app.FeedStore
+import com.github.jetbrains.rssreader.app.initKoin
+import kotlinx.coroutines.MainScope
+import kotlinx.coroutines.cancel
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.launch
+import org.koin.core.component.KoinComponent
+import org.koin.core.component.get
+
+fun main() {
+    console.log("loading shared library")
+}
+
+// KoinComponent is an interface therefore it is not exportable
+// I don't know how to ignore the warning.. it works tho
+
+@ExperimentalJsExport
+@JsExport
+object RssReaderJsViewModel : KoinComponent {
```

Comment:
I think you could move all kotlin part from webApp to shared module.
So webApp will be pure js application which uses shared module compiled as js bundle

Reply:
Yes it actually works ☑️